tests/test_updater_consistent_snapshot.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+
+# Copyright 2021, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+"""Test ngclient Updater consistent snapshot"""
```

Comment:
this could be better (we're testing consistent snapshot in all client tests, hopefully). This file seems to test 
* that the client works with all combinations of consistent_snapshot and prefix_targets_with_hash
* that expected URLs are being fetched with all of these combinations